ietf/api/views.py

```diff
@@ -202,6 +203,27 @@ def err(code, text):
             else:
                 send_account_creation_email(request, email)
                 response += ", Email sent"
+
+            # handle nomcom volunteer
+            if data['is_nomcom_volunteer'] and object.person:
+                # registrants for the fall meeting will be volunteering for the
+                # following year's nomcom
```

Comment:
That's true for the summer meeting too (the nomcom selection process is run by then usually).

Instead I think we should discuss when the registration system should offer volunteering for nomcom - it may only want to do so only for the spring meeting. Or only if there is a nomcom configured that is accepting volunteers (and we can make the assumption that there would never be two nomcoms accepting volunteers at the same time.).